MapboxSearch/autofill/src/test/java/com/mapbox/search/autofill/TypeObjectCreator.kt

```diff
@@ -28,7 +29,13 @@ internal object TypeObjectCreator {
         )
 
         listOf(
-            AddressAutofillSuggestion("name", "formattedAddress", addressComponents, null)
+            AddressAutofillSuggestion(
+                name = "name",
+                formattedAddress = "formattedAddress",
+                coordinate = Point.fromLngLat(10.0, 15.0),
+                address = addressComponents,
+                underlying = null
```

Comment:
Is this line added intentionally?

Reply:
Yes, this is a test (note `.../src/test/...` in the path)